cfme/fixtures/rest_api.py

```diff
@@ -7,3 +7,8 @@
 @pytest.fixture(scope="function")
 def rest_api():
     return store._current_appliance[-1].rest_api
+
+
+@pytest.fixture(scope="module")
+def rest_api_module():
```

Comment:
This was a little confusing, since it looks like it would return a module. Would you mind calling it `rest_api_modscope`, or something similar?